tools/helper.py

```diff
@@ -48,6 +49,20 @@ def check_if_node_is_registered(skale, node_id):
     return True
 
 
+def check_required_balance(skale):
+    address = skale.wallet.address
+    eth_bal_before_tx = skale.web3.eth.getBalance(address)
+    if eth_bal_before_tx < MIN_ETH_AMOUNT:
+        logger.info(f'ETH balance: {eth_bal_before_tx} is less than {MIN_ETH_AMOUNT}')
+        # TODO: notify SKALE Admin
```

Comment:
API for notifications is already available

Reply:
great!!!!